client/reader/search-stream/style.scss

```diff
@@ -34,6 +34,11 @@
 	width: 830px;
 }
 
+/* clamp to 3 lines for search-rec excerpts */
+ .is-reader-page .search-stream .reader-related-card-v2__excerpt.post-excerpt {
+	max-height: 15px * 1.6 * 3;
```

Comment:
should we also include the `long-content-fade` here, like we do for the related cards in the main stream?